README.md

```diff
@@ -107,6 +107,47 @@ The `check` task will check formatting (in addition to the other normal checks),
 
 The `check` task will ensure that license headers are properly applied, and `licenseFormat` will apply headers for you.
 
+## Releasing a version
+
+### Checking artifacts locally
+
+To see the artifacts that would be released, build the relevant artifacts locally using a fake version `foo`:
+
+```
+./gradlew publishSonatypePublicationToLocalDebugRepository -Pversion=foo
+tree */build/repos/localDebug
+```
+
+### Maven Central requirements
+
+To publish to Maven Central, you'll need the `sonatypeUsername` and `sonatypePassword` Gradle properties set (`~/.gradle/gradle.properties` is typically where people put these). You'll also need GPG set up for the [signing plugin](https://docs.gradle.org/current/userguide/signing_plugin.html), as per Sonatype's [requirements](https://central.sonatype.org/publish/requirements/gpg/). All told, your properties should have:
+
+
+```
+sonatypeUsername = ...
+sonatypePassword = ...
```

Comment:
where can I create the account for this? I didn't see it on https://www.sonatype.com/

Reply:
Sonatype has a process for accounts that can publish to maven central; https://central.sonatype.org/publish/ has the details